hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -347,7 +347,6 @@ public class HRegion implements HeapSize, PropagatingConfigurationObserver, Regi
   final LongAdder compactionsFailed = new LongAdder();
   final LongAdder compactionNumFilesCompacted = new LongAdder();
   final LongAdder compactionNumBytesCompacted = new LongAdder();
-  final LongAdder compactionsQueued = new LongAdder();
```

Comment:
It seems a bit strange that we only move this field to HStore but still keep other compaction related metrics in HRegion...

Reply:
Yeah, you are right, i thought about it too, but did not find a better way, maybe all that metrics should be moved to HStore?

Reply:
I think moving compaction metrics to HStore would be better.

Reply:
> I think moving compaction metrics to HStore would be better.

I can move the other compaction metrics in a follow-on issue.

Reply:
Then is it possible to not include the metrics moving but still fix the problem here? It is not a good idea to land a half done work, so I prefer either we do not move any of them, or move them all at once.